### PR TITLE
Add bob.bio.video recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
This depends on https://github.com/conda-forge/staged-recipes/pull/495

**Comment by ccordoba12** [comment]
@183amir, just out of curiosity I'd like to ask you: what's the meaning of the `bob` or `bob.bio` namespace?

**Comment by 183amir** [comment]
[Bob](https://idiap.github.io/bob/) is a signal-processing and machine learning toolbox originally developed by the Biometrics Group at Idiap, in Switzerland.
`bob.bio` are packages that are written to help running `biometric` experiments.

**Comment by 183amir** [comment]
@ocefpaf since you asked in another place about skipping windows build on `bob` packages, I have said to skip builds here but it runs and fails Windows builds anyway.

**Comment by ocefpaf** [comment]
@183amir the issue is another bug that is already fixed in latest `conda-build-all` (https://github.com/SciTools/conda-build-all/issues/41). However, latest `conda-build-all` is not available for all matrix options on Windows b/c of another bug (https://github.com/conda-forge/conda-build-all-feedstock/issues/12).

A fix is under way, but I don't see a problem merging this with a failing AppVeyor.

**Comment by 183amir** [comment]
If there isn't any problems, please merge this.

**Comment by ocefpaf** [comment]
> If there isn't any problems, please merge this.

Sure. I am merging one at a time to avoid accumulating many recipes when Travis-CI start failing. I will merge this as soon as `pyjwt` clears.

File: recipes/bob.bio.video/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    - bob.bio.video.test.dummy
+    - bob.bio.video.utils
+
+  requires:
+    - nose
+    - bob.db.mobio
+    - bob.db.youtube
+
+about:
+  home: https://www.github.com/<YourInstitution>/<YourPackage>
```
Comment (ocefpaf) [inline recipes/bob.bio.video/meta.yaml]: Is that correct @183amir?

Comment (183amir) [inline recipes/bob.bio.video/meta.yaml]: oops sorry it's coming from it's PyPI info: https://pypi.python.org/pypi/bob.bio.video

Comment (183amir) [inline recipes/bob.bio.video/meta.yaml]: Fixed!

**Pull request merged**